doc/releases/changelog-0.30.0.md

```diff
@@ -211,10 +412,10 @@
 * Added `Operation.__truediv__` dunder method to be able to divide operators.
```

Comment:
🤔? Does this just mean, say, `A / B` is possible?

Reply:
Check out here: https://github.com/PennyLaneAI/pennylane/pull/3749/files#diff-ed5f9c0de17892f74e5119e2a032fe277eaa330ec4df0c249c7a75af7be6585eR1429

It allows you to divide _by a scalar_.

Added a clarification here: b6f91bd7ddb55a531c7d98e450cbc3b929d9dba6